ViteMaDose/Views/Home/Cells/HomeStatsCell.swift

```diff
@@ -107,7 +106,7 @@ struct HomeCellStatsViewData: HomeStatsCellViewDataProvider, Hashable {
                 withConfiguration: UIImage.SymbolConfiguration(pointSize: 18, weight: .bold)
             )?.withTintColor(.label, renderingMode: .alwaysOriginal)
 
-            let fullString = NSMutableAttributedString(string: "Ouvrir la carte des centres ")
+            let fullString = NSMutableAttributedString(string: Localization.Home.open_map)
```

Comment:
Request change. Consistency: Add "+ String.space" and remove the trailing space from the original String.